time_weighted_average_price/attest_fn_call/Makefile

```diff
@@ -17,37 +15,39 @@ tmp/x.wasm: main.go $(wildcard as/*.go) $(wildcard price/*.go) | tmp check
         tinygo/tinygo:0.31.2 \
         tinygo build -o tmp/x.wasm -target=wasi ./...
 
+.PHONY: build
 build: tmp/x.wasm
 
+.PHONY: prev
+prev: | check
```

Comment:
In previous PRs, I had not looked too deeply at the structure of the makefile, so this issue may be in more places than just this one. If we are doing ordered targets, it looks like there are either some bugs in the dependencies here or we are making things more complex than needed.  Specifically, because `check` has no output, it will always run, in which case, we don't need order dependencies.  Alt, if we don't want check to run at every run, we should output a file into tmp.

Reply:
Good point. I cleaned up the targets